brianlong/www.validators.app
# 186389686 tweak logos

#### What's this PR do?
- Tweaks default validator logos 
- Prevents from assigning default url from keybase

#### How should this be manually tested?
- Remove default avatars from db:
```
Validator.where(avatar_url: "https://keybase.io/images/no-photo/placeholder-avatar-180-x-180@2x.png").update_all(avatar_url: nil)
Validator.where(avatar_url: "https://keybase.io/images/no-photo/placeholder-avatar-180-x-181@2x.png").update_all(avatar_url: nil)
```
- Run `rake assets:clobber ` if necessary
- Then run standard deamons (gather+score) to download avatars from solana CLI 
- Then run keybase script to download the rest of avatars: `rails r script/validators_get_avatar_url.rb`
- Go through all pages where we display avatar: validators index, validator details page, data centers, stake pools and a map on homepage (run `rails r daemons/leader_stats_mainnet_update_daemon.rb`)
- Confirm that avatars on all of these pages are correctly displayed
- Confirm that validator name of current leader on the map is correctly displayed (full name or truncated identity), and that validator details page is opened when you click on leader name or leader avatar

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186389686)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
